src/libraries/System.Text.RegularExpressions/gen/Resources/Strings.resx

```diff
@@ -0,0 +1,318 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
can we not reuse the existing resx?

Reply:
How? i.e. is there a mechanism for including an additional resx file and having them merged, or something like that?

Reply:
I meant just use the existing one - of course not all the strings apply. Perhaps that would get complex given this one is localized.
No I don't know a way to include one in another.

Reply:
> I meant just use the existing one

And add the generator specific strings into the existing one?

Reply:
If that's less wasteful than the duplication. Feel free to ignore.